### DISCUSSION TITLE
New product icon needed for Alarms - I have SVG file

### DISCUSSION THREAD

**Discussion by eflann** [twilio-labs/paste#1862] [Request: Icon] [answered]
Can you please add the bell icon to paste so that it can be used for the new Alarms Product in the Nav?

I have the bell svg file. (I can't attach it to this issue because github doesn't support svgs.)

<img width="1440" alt="Screen Shot 2021-09-13 at 9 28 33 AM" src="https://user-images.githubusercontent.com/16602449/133170361-c384e366-2272-4347-82f2-3db546715bd8.png">

**Comment by sang-yeo** [comment]
We double post same topic. Streamline icon link: https://app.streamlinehq.com/icons/streamline-light/interface-essential/alert/alarm-bell

**Comment by nkrantz** [comment]
We've added this icon to our next sprint and we'll let you know when it's been added to the system!

**Reply by sang-yeo** [reply]
Awesome! thank you

**Comment by andioneto** [comment]
@sang-yeo In the spirit of ruthless prioritization, we had to push this icon addition to our next sprint, which starts next Wednesday. Thanks for your patience and for helping us make Paste better. We will let you know as soon as the icon is available. 🙇‍♀️

**Reply by eflann** [reply]
Has this been completed? Not seeing the icon.

**Reply by SiTaggart** [reply]
It just got into PR today https://github.com/twilio-labs/paste/pull/1957

**Comment by loreina** [comment, marked as answer]
Hey @eflann, thanks for being patient about this. The Alarms icon is now live in Figma as `logo/product/Alarms` and in code as `ProductAlarmsIcon`. ⏰ 🥳

**Reply by sang-yeo** [reply]
Super! Thanks
